packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -432,7 +434,7 @@ Posts.addView("recentDiscussionThreadsList", terms => {
   return {
     selector: {
       baseScore: {$gt:0},
-      hideFrontpageComments: false,
+      hiddenRelatedQuestion: viewFieldAllowAny,
```

Comment:
Why did you get rid of `hideFrontpageComments` here? Seems like we still want that functionality?

Reply:
Oh, whoops, I totally misread that as hiddenRelatedQuestions.

Reply:
Fixed